content/300-guides/200-deployment/201-serverless/400-deploy-to-aws-lambda.mdx

```diff
@@ -1,301 +1,217 @@
-
-The exact Serverless Framework commands that you must use depend on the way that you install the Serverless Framework CLI.
-
-- If you install the Serverless Framework CLI globally, then you can use the commands exactly as shown in this guide.
-
-- If you install the Serverless Framework CLI in your project, then you must prepend `npx` to the commands in this guide. For example:
-
-  ```terminal wrap
-  npx serverless config credentials --provider aws --key AWS_ACCESS_KEY_ID  --secret AWS_SECRET_ACCESS_KEY
-  ```
+This guide assumes you have an existing project using both Serverless Framework and Prisma. For a guide on how to get started with Serverless Framework, we recommend [Setting Up Serverless Framework With AWS](https://www.serverless.com/framework/docs/getting-started).
 
-## Prisma workflow
+## Binary targets in <inlinecode>schema.prisma</inlinecode>
 
-In this guide, you do the following:
+The Prisma schema should contain the following in the `generator` block:
 
-- Use Prisma and a Prisma schema file (`schema.prisma`) to create a schema for the database
-- Use `prisma db push` to push this Prisma schema to the database
-
-In this guide, you use `prisma db push` to push the initial Prisma schema to the database. For a real project, you would typically start with a development database and use `prisma db push` as we suggest here. You can continue with `prisma migrate dev`, which generates a migration script for each change that you make to the Prisma schema. With multiple migration scripts, you build a migration history for your database and that makes it easy to later apply the changes to a production database with `prisma migrate deploy`. For more information, see [Prisma Migrate](/concepts/components/prisma-migrate). <br></br><br></br>If you use Supabase to host your database, then when you use `prisma migrate` you must set up a shadow database. For details, see the [Supabase docs](https://supabase.com/docs/guides/integrations/prisma).
-
-## 1. Download the example project
-
-To start, download the example project. The project includes the configuration that you need to complete this guide.
-
-### Steps
-
-1. Open a terminal window and navigate to a location of your choice. Use the following commands to create a directory for the project and download the example code:
-
-   ```terminal wrap
-   mkdir prisma-aws-lambda
-   cd prisma-aws-lambda
-   curl https://codeload.github.com/prisma/prisma-examples/tar.gz/latest | tar -xz --strip=3 prisma-examples-latest/deployment-platforms/aws-lambda/
-   ```
-
-2. Install the dependencies:
-
-   ```terminal
-   npm install
-   ```
-
-<!-- tar strip folder is a concatenation of the REPOSITORY-BRANCH/REF, e.g. prisma-examples-latest -->
-
-### Result
-
-Your directory contains the following files:
-
-```terminal no-copy
-README.md
-handlers
-node_modules
-package-lock.json
-package.json
-prisma
-schema.sql
-serverless.yml
+```prisma
+binaryTargets = ["native", "rhel-openssl-1.0.x"]
```

Comment:
Based on Niko's feedback, it could be worth adding a note on what `rhel-openssl-1.0.x` is (Not a blocker but for a future PR)

Reply:
https://github.com/prisma/docs/issues/5186